ngraph/core/src/op/region_yolo.cpp

```diff
@@ -69,15 +70,19 @@ void op::RegionYolo::validate_and_infer_types() {
                 output_shape.push_back(input_shape[i]);
             }
             for (int64_t i = m_axis; i < end_axis + 1; i++) {
-                flat_dim *= input_shape[i];
+                if (input_shape[i].is_dynamic()) {
```

Comment:
I have a better way to formulate this piece of code. Will do it in the separate pr.

Reply:
can you provide link to PR?